src/Compilers/CSharp/Test/Syntax/Parsing/ParsingTests.cs

```diff
@@ -1,6 +1,6 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-// #define PARSING_TESTS_DUMP
+//#define PARSING_TESTS_DUMP
 
```

Comment:
This is the only change in this file. Consider reverting it to the previous state

Reply:
This is actually intentional, so that the file corresponds to what VS produces when you uncomment-line and then comment-line it.